gunicorn.config.py

```diff
@@ -20,7 +20,7 @@
 # and serve the requests before the downstream timeout.
 timeout = 15
 
-grateful_timeout = 120
+graceful_timeout = 600
```

Comment:
is `600` the correct value? if queries run ~10 mins, there should be an additional grace period. Sounds more like `630` to me then
```suggestion
graceful_timeout = 630
```